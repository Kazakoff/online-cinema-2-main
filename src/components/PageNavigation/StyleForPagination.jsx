import React from 'react';

export const StyleForPagination = (props) => {

  return (
    <div className='d-flex justify-content-center' style={{ marginBottom: '12px' }}>
      {props.children}
    </div>
  )
}
