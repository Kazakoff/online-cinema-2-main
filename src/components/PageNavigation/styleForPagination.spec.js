import React from 'react'
import { StyleForPagination } from './StyleForPagination'

const setUp = () => shallow(<StyleForPagination />);

describe('StyleForPagination component', () => {
    it('should render StyleForPagination component', () => {
        const component = setUp()
        expect(component.debug()).toMatchSnapshot()
    })
})