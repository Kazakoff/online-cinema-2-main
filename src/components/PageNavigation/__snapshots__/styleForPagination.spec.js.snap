// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyleForPagination component should render StyleForPagination component 1`] = `"<div className=\\"d-flex justify-content-center\\" style={{...}} />"`;
